demo/style.css

```diff
@@ -0,0 +1,17 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600&display=swap');
```

Comment:
Suggestion: whilst this is a demo page only for devs, in general we should avoid loading fonts from Google Fonts. We don't do this on our public sites as they violate GDPR in some EU countries, as an example.

Sticking to a native font stack is fine for this demo I think.